test/recipe-test.js

```diff
@@ -3,13 +3,21 @@ import { expect } from 'chai'
 import Recipe from '../src/recipe'
 import data from '../src/data/recipe-data'
 
+import ingredientsData from '../src/data/ingredient-data'
+
 describe('Recipe', function() {
   let recipe
   let recipeInfo
+  let recipe2Info
```

Comment:
Nitpicky comment but I'd swap lines 11 and 12 to be consistent with the order you're using for recipe & recipeInfo